web/frontend/src/components/slam/index.svelte

```diff
@@ -3,7 +3,7 @@
 
   import * as THREE from 'three';
   import { onMount } from 'svelte';
-  import { slamApi, SlamClient, type Pose, type ServiceError } from '@viamrobotics/sdk';
+  import { slamApi, motionApi, SlamClient, MotionClient, type Pose, type ServiceError } from '@viamrobotics/sdk';
```

Comment:
@ethanlook @micheal-parks @njooma 
I need to use `MotionClient` for the `motion.getPlan` API endpoint as that endpoint returns a deeply nested structure & the `robotClient.motionService.getPlan` function returns a structure that is not human readable:
[robotClient.motionService.getPlan.json](https://github.com/viamrobotics/rdk/files/14016392/robotClient.motionService.getPlan.json)

whereas `MotionClient.getPlan` does return data that is human readable:
[motionClient.getPlan.json](https://github.com/viamrobotics/rdk/files/14016396/motionClient.getPlan.json)

As a result, motion APIs are now accessed on this page by both `robotClient.motionService` & `MotionClient` objects. That seems nonideal to me (as 2 objects are doing the same job) but I don't see a way around it without some guidance on how this should work.

Reply:
I think that we always want to use the named `MotionClient` that we instantiate, not the `robotClient.motionService`. Could the `web/frontend/src/api/motion.ts` use the shared client? Let me know if you want to chat in person.

Reply:
Discussed offline:
moveOnMapNew is not yet defined on the MotionClient (this got missed) so we are going to move forward with the current sate of this PR.

As part of https://viam.atlassian.net/browse/RSDK-6431 which will add `moveOnMap`'s final implementation to the MotionClient I'll plan on updating the slam index card to only use the MotionClient, rather than the robotClient.motoinService.